src/toast/pipeline_tools/todground.py

```diff
@@ -0,0 +1,496 @@
+# Copyright (c) 2019 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import argparse
+import dateutil.parser
+import os
+
+import healpy as hp
+import numpy as np
+
+from ..timing import function_timer, Timer
+from ..utils import Logger, Environment
+from ..weather import Weather
+
+from ..tod import OpSimAtmosphere, atm_available_utils
+
+
+# Telescope, Site and CES are small helper classes for building
+# ground observations
+
+
+def name2id(name, maxval=2 ** 16):
+    """ Map a name into an index.
+
+    """
+    value = 0
+    for c in name:
+        value += ord(c)
+    return value % maxval
+
+
+class Telescope(object):
+    def __init__(self, name):
+        self.name = name
+        self.id = name2id(name)
+        return
+
+
+class Site(object):
```

Comment:
no need to subclass `object` in Python 3

Reply:
Thanks. Doesn't hurt either.

Reply:
It is a easy "replace with empty string" fix that removes code, I would do it

Reply:
Agreed.